henson-processor/src/test/java/com/f2prateek/dart/henson/processor/HensonNavigatorGeneratorTest.java

```diff
@@ -278,4 +283,110 @@
         .and()
         .generatesSources(expectedSource1);
   }
+
+  @Test public void samePackage() throws Exception {
```

Comment:
we should add unit tests for the code you added. Not a generator test to test a lookup method in package name..